### PR TITLE
Remove namespace_user_id and namespace_org_id from external services

### PR DESCRIPTION
**Pull request by miveronese**
This PR removes `namespace_user_id` and `namespace_org_id` from `graphqlbackend/external_services.go` as we stopped supporting  namespaced external services.

It can be merged after this other PR.
https://github.com/sourcegraph/sourcegraph/pull/43014

## Test plan

Confirmed that tests are passing locally.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | -0.00% (-0.02 kb) | -0.00% (-0.02 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/mv/clean-namespaced-extsvcs-code/compare-report.html;tab=live_object) for a full comparison between the commits 90d6776 and c35bcec or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: cmd/frontend/graphqlbackend/external_services.go [hunk 1]
```diff
@@ -262,14 +244,7 @@ func (r *schemaResolver) ExternalServices(ctx context.Context, args *ExternalSer
 	}
 
 	opt := database.ExternalServicesListOptions{
-		// 🚨 SECURITY: When both `namespaceUserID` and `namespaceOrgID` are not
-		// specified we need to explicitly specify `NoNamespace`, otherwise site
-		// admins will be able to list all user code host connections that are not
-		// accessible when trying to access them individually.
-		NoNamespace:     namespaceUserID == 0 && namespaceOrgID == 0,
```
Comment (mrnugget) [inline cmd/frontend/graphqlbackend/external_services.go:269]: Should we keep this until we've removed the notion of "external service namespace" from the database/store layer and are confident that it's not a problem?

_Technically_ I think it's already fine, but I'm still a bit scared and would rather remove this together with removals in the store layer.

**Review by mrnugget — APPROVED** [review]
VERY NICE!!

File: cmd/frontend/graphqlbackend/external_services.go (hunk 1 above)
Comment (miveronese) [reply cmd/frontend/graphqlbackend/external_services.go:269]: You are right! -- better to be on the safer side here.
 I will add this part back and remove it in the next PR along with the DB layer removals.

**Pull request merged**